apps/docs/src/app/core/component-docs/info-label/info-label-header/info-label-header.component.html

```diff
@@ -0,0 +1,13 @@
+<header>Info Label</header>
+<description>
+    <p>Info labels are used to indicate status. Colors, generally in combination with text, numeric and icon, are used to easily
```

Comment:
I think this description needs to be improved.

> Colors, generally in combination with text, numeric and icon, are used to easily highlight.

Highlight what?

> Black is used to indicate default or inactive status  

What "Black"?

> Diffrent color options are availabel to define the status of the object

Typo

> Info Label is a small non-interactive numeric or text based control

I feels like you should start from that

> Diffrent color options are availabel to define the status of the object

What object?